PryntTrimmerView/Classes/Trimmer/PryntTrimmerView.swift

```diff
@@ -256,14 +259,18 @@ import UIKit
     }
 
     private func updateLeftConstraint(with translation: CGPoint) {
-        let maxConstraint = max(rightHandleView.frame.origin.x - handleWidth - minimumDistanceBetweenHandle, 0)
-        let newConstraint = min(max(0, currentLeftConstraint + translation.x), maxConstraint)
+        let maxConstraint = rightHandleView.frame.origin.x - minimumDistanceBetweenHandle - handleWidth
+        let minConstraint = max(0, rightHandleView.frame.origin.x - maximumDistanceBetweenHandle - handleWidth )
+        let theWantedNewX = currentLeftConstraint + translation.x
```

Comment:
Maybe we should call this something like desiredX, or at least stick with the naming conventions of the original authors rather than introducing our convention of "theLocalVariableName."